source/includes/php/useElement.md

```diff
@@ -0,0 +1,302 @@
+# Manage a single Element
+
+## Original values
+
+> Get original values
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('yourApiToken');
+
+$product = \Shoprunback\Elements\Product::retrieve('1f27f9d9-3b5c-4152-98b7-760f56967dea');
+
+echo $product->label; // Prints: 'My label'
+
+$product->label = 'New label';
+
+echo $product->label; // Prints: 'New label'
+
+echo $product->_origValues->label; // Prints 'My label'
+
+$product->refresh();
+
+echo $product->label; // Prints 'My label'
+```
+
+You can get the original values of an Element whenever you want by getting the `_origValues` attribute.
+
+**Never edit the** `_origValues`, there are a landmark.
+
+If you want to reset an element with its original values, use `$element->refresh()` (makes an API call)
+
+<aside class="warning">
+When the <b>element is saved</b>, the <b>origValues</b> will be <b>filled with the new values!</b>
+</aside>
+
+## Get different attributes
+
+> Different ways to get important attributes
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('yourApiToken');
+
+$product = new \Shoprunback\Elements\Product();
+
+$product->label = 'My label';
+$product->custom = 'My custom param';
+$product->brand = new \Shoprunback\Elements\Brand();
+
+var_dump($product->getAllAttributes()); // Prints 'label', 'custom' and 'brand' with their values
```

Comment:
can you actually write the result of the var_dump ?

Reply:
No, too big, it would take like 20 lines